spec/models/program_session_spec.rb

```diff
@@ -1,5 +1,145 @@
 require 'rails_helper'
 
-RSpec.describe ProgramSession, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+describe ProgramSession do
+
+  let(:proposal) { create(:proposal_with_track, :with_two_speakers) }
+
+  it "responds to .create_from_proposal" do
+    expect(ProgramSession).to respond_to(:create_from_proposal)
+  end
+
+  describe ".create_from_proposal" do
+
+    it "program session and proposal share the same title" do
```

Comment:
The describe identifies the thing or process being tested. With these "it" blocks, try "finishing the sentence" to describe the behavior: (describe) #create_from_proposal (it) creates a new program session with the same title as the given proposal. Sorry should have mentioned that yesterday when I commented on the describe, since this comment goes with that.